.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,43 @@
+name: Bug Report
+description: Create a report to help us improve
+title: "[Bug] "
+labels: [':bug: Bug']
+body:
+- type: input
```

Comment:
Separate this input into two:

* Plugin Name (id: pluginName) 
	* Use the *type*: dropdown
	* Use as possibles *options*: @nxrocks/spring-boot, @nxrocks/flutter, @nxrocks/quarkus
* Plugin Version (id: pluginVersion)
	* Add *placeholder* with value: `for e.g. v1.0.0`